scripts/o3de/o3de/project_properties.py

```diff
@@ -31,22 +31,31 @@ def get_project_props(name: str = None, path: pathlib.Path = None) -> dict:
 def _edit_gem_names(proj_json: dict,
                     new_gem_names: str or list = None,
                     delete_gem_names: str or list = None,
-                    replace_gem_names: str or list = None):
+                    replace_gem_names: str or list = None,
+                    is_optional: bool = False):
```

Comment:
Did you test activating/deactivating Gems in the Project Manager after making this change? Would be good as a sanity check to make sure nothing is broken there.

Reply:
Good idea, I'll try it and update when I inevitably find an error ;)

Reply:
I ran into an issue that I commented about below, but other than that, enabling/disabling is working.